<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_Title">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>88</red>
            <green>88</green>
            <blue>88</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>88</red>
            <green>88</green>
            <blue>88</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>118</red>
            <green>116</green>
            <blue>108</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>88</red>
            <green>88</green>
            <blue>88</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>88</red>
            <green>88</green>
            <blue>88</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Image Loader</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>966</width>
             <height>119</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_LoadImage">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Load Image</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/Icons/Icons/icons8-add-image-32.png</normaloff>:/Icons/Icons/icons8-add-image-32.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_drawPolygon">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Draw polygon</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/Icons/Icons/icons8-polygon-filled-50.png</normaloff>:/Icons/Icons/icons8-polygon-filled-50.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_ResetImage">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Reset Coordinates</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/Icons/Icons/icons8-synchronize-filled-50.png</normaloff>:/Icons/Icons/icons8-synchronize-filled-50.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_clearImage">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Clear Image</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/Icons/Icons/icons8-delete-filled-50.png</normaloff>:/Icons/Icons/icons8-delete-filled-50.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_saveCordinatesInDB">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Save Cordinates in DB</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/Icons/Icons/icons8-database-50.png</normaloff>:/Icons/Icons/icons8-database-50.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_SaveImage">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Save Image</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/Icons/Icons/icons8-save-as-filled-50.png</normaloff>:/Icons/Icons/icons8-save-as-filled-50.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_image"/>
    <addaction name="action_clearImage"/>
    <addaction name="actionSave_Image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSet_penColor"/>
    <addaction name="actionSet_penWidth"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>false</italic>
    </font>
   </property>
  </widget>
  <action name="actionLoad_image">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Icons/icons8-add-image-32.png</normaloff>:/Icons/Icons/icons8-add-image-32.png</iconset>
   </property>
   <property name="text">
    <string>Load image</string>
   </property>
  </action>
  <action name="action_clearImage">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Icons/icons8-delete-filled-50.png</normaloff>:/Icons/Icons/icons8-delete-filled-50.png</iconset>
   </property>
   <property name="text">
    <string>Clear Image</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Icons/icons8-save-as-filled-50.png</normaloff>:/Icons/Icons/icons8-save-as-filled-50.png</iconset>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Icons/icons8-exit-50.png</normaloff>:/Icons/Icons/icons8-exit-50.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSet_penColor">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Icons/icons8-compose-50.png</normaloff>:/Icons/Icons/icons8-compose-50.png</iconset>
   </property>
   <property name="text">
    <string>Set Pen Color</string>
   </property>
  </action>
  <action name="actionSet_penWidth">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Icons/icons8-width-50.png</normaloff>:/Icons/Icons/icons8-width-50.png</iconset>
   </property>
   <property name="text">
    <string>Set Pen Width</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Icons/icons8-about-50.png</normaloff>:/Icons/Icons/icons8-about-50.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Icons/icons8-qt-50.png</normaloff>:/Icons/Icons/icons8-qt-50.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
